qa/tasks/vstart_runner.py

```diff
@@ -865,7 +871,7 @@ def raw_cluster_cmd(self, *args, **kwargs):
         return stdout string
         """
         proc = self.controller.run(args=[os.path.join(BIN_PREFIX, "ceph")] + \
-                                        list(args), **kwargs)
+                                        list(args), **kwargs, stdout=BytesIO())
```

Comment:
Why BytesIO ? And the next line will switch it to str.

Reply:
`raw_cluster_cmd()` in ceph_manager uses BytesIO, I'll change that to StringIO instead - https://github.com/ceph/ceph/blob/master/qa/tasks/ceph_manager.py#L1363.

Reply:
Fixed.